src/IERC20.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity >=0.6.2;
+
+interface IERC20 {
+  function totalSupply() external view returns (uint);
+
+  function balanceOf(address account) external view returns (uint);
+
+  function transfer(address recipient, uint amount) external returns (bool);
+
+  function allowance(address owner, address spender) external view returns (uint);
+
+  function approve(address spender, uint amount) external returns (bool);
+
+  function transferFrom(address sender, address recipient, uint amount) external returns (bool);
+
+  function symbol() external view returns (string memory);
+
+  event Transfer(address indexed from, address indexed to, uint value);
+  event Approval(address indexed owner, address indexed spender, uint value);
+
+  function decimals() external view returns (uint8);
+
+  function name() external view returns (string memory);
```

Comment:
Its just moving stuff around, and adding this method? :)

Reply:
Yes I noticed that it was missing, even though it is part of the [ERC20](https://eips.ethereum.org/EIPS/eip-20) standard